src/_js/lib/HeaderLinker.js

```diff
@@ -1,4 +1,4 @@
-const init = function() {
+$(document).on('page.didUpdate', function() {
```

Comment:
til thats an event

Reply:
It's a thing I added myself that fires after the dynamic update. It's not native.